docs/ReleaseNotes/Kubernetes-Operator-for-PXC-RN1.17.0.md

```diff
@@ -0,0 +1,107 @@
+# Percona Operator for MySQL based on Percona XtraDB Cluster 1.17.0 ({{date.1_17_0}})
+
+[Installation](../System-Requirements.md#installation-guidelines){.md-button}
+
+## Release Highlights
+
+This release of Percona Operator for MySQL based on Percona XtraDB Cluster includes the following new features and improvements:
+
+### Improved observability for HAProxy and ProxySQL
+
+Get insights into the HAProxy and ProxySQL performance by connecting to their statistics pages. Use the `cluster-name-haproxy:8084` and `cluster-name-proxysql:6070` endpoints to do so. Learn about other available ports in the [documentation](../haproxy-conf.md).
+
+### Improved cluster load management during backups
+
+If parallel backups overload your cluster, you can turn off parallel execution to prevent this. Previously, this meant that you could only run one backup at a time - no new backups could start until the current one was finished. Now, the Operator queues backups and runs them one after another automatically. You can fine-tune the backup sequence by setting the start time for all backups or for a specific on-demand one using the [`spec.backup.startingDeadlineSeconds`](../operator.md#backupstartingdeadlineseconds) Custom Resource option. This provides greater control over backup operations.
+
+Another improvement is for the case when your database cluster becomes unhealthy, for example, when a Pod crashes or restarts. The Operator suspends running backups to reduce the cluster's load. Once the cluster recovers and reports a Ready status, the Operator resumes the suspended backup. To further offload the cluster during an unhealthy state, you can configure how long a backup remains suspended by using the [`spec.backup.suspendedDeadlineSeconds`](../operator.md#backupsuspendeddeadlineseconds) Custom Resource option. If this time expires before the cluster recovers, the backup is marked as "failed."
+ 
+### Monitor PMM Client health and status
+
+Percona Monitoring and Management (PMM) is a great tool to [monitor the health of your database cluster](../monitoring.md). However, you may also want to know if PMM itself is healthy. Now you can do it using probes - a Kubernetes diagnostics mechanism to check the health and status of containers. Use the[ `spec.pmm.readinessProbes.*`](../operator.md#pmmreadinessprobesinitialdelayseconds) and [`spec.pmm.livenessProbes.*`](../operator.md#pmmlivenessprobesinitialdelayseconds) Custom Resource options to enable Readiness and Liveness probes for PMM Client.
```

Comment:
> Now you can do it using probes - a Kubernetes diagnostics mechanism to check the health and status of containers. Use the[ `spec.pmm.readinessProbes.*`](../operator.md#pmmreadinessprobesinitialdelayseconds) and [`spec.pmm.livenessProbes.*`](../operator.md#pmmlivenessprobesinitialdelayseconds) Custom Resource options to enable Readiness and Liveness probes for PMM Client.

You can't enable or disable these probes you can change configuration for these probes.  They are enabled by default.

Reply:
Updated